build/containers/Dockerfile.onvif-video-broker

```diff
@@ -14,13 +14,25 @@ RUN find . && dotnet build "onvif-video-broker.csproj" -c Release -o /app/build
 
 FROM build AS publish
 ARG DOTNET_PUBLISH_RUNTIME
-RUN echo "Publishing to: ${DOTNET_PUBLISH_RUNTIME}"
-RUN dotnet publish -r ${DOTNET_PUBLISH_RUNTIME} "onvif-video-broker.csproj" -c Release -o /app/publish
+RUN echo "Publishing to: ${DOTNET_PUBLISH_RUNTIME}" && \
+    dotnet publish -r ${DOTNET_PUBLISH_RUNTIME} "onvif-video-broker.csproj" -c Release -o /app/publish
 
-FROM ghcr.io/deislabs/akri/opencvsharp-build:${OUTPUT_PLATFORM_TAG} AS final
+FROM ghcr.io/kate-goldenring/opencvsharp-build:0.0.7-amd64 AS final
 ARG OUTPUT_PLATFORM_TAG
 RUN echo "Output base image: ghcr.io/deislabs/akri/opencvsharp-build:${OUTPUT_PLATFORM_TAG}"
 
+# Install OpenCVSharpEntern dependencies
+RUN apt-get update && apt-get -y install --no-install-recommends \
+    libgtk2.0-dev \
```

Comment:
should these be installed in opencvsharp-build?

Reply:
After bringing over the library, the linker couldn't find the dependencies, so I installed the ones that were missing:
```
Step 19/26 : RUN ldd /usr/lib/libOpenCvSharpExtern.so
 ---> Running in b2fccf8b0137
        linux-vdso.so.1 (0x00007fffd51fb000)
        libharfbuzz.so.0 => not found
        libgtk-x11-2.0.so.0 => not found
        libgdk-x11-2.0.so.0 => not found
        libpangocairo-1.0.so.0 => not found
        libatk-1.0.so.0 => not found
        libcairo.so.2 => not found
        libgdk_pixbuf-2.0.so.0 => not found
        libgio-2.0.so.0 => not found
        libpangoft2-1.0.so.0 => not found
        libpango-1.0.so.0 => not found
        libgobject-2.0.so.0 => not found
        libglib-2.0.so.0 => not found
        libfontconfig.so.1 => not found
        libfreetype.so.6 => not found
        libgthread-2.0.so.0 => not found
        libtesseract.so.4 => not found
        libdc1394.so.22 => not found
        libavcodec.so.58 => not found
        libavformat.so.58 => not found
        libavutil.so.56 => not found
        libswscale.so.5 => not found
        libavresample.so.4 => not found
        libpng16.so.16 => not found
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007ff70a725000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007ff70a720000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007ff70a6ff000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007ff70a6f5000)
        libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007ff70a571000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007ff70a3ec000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007ff70a3d2000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007ff70a211000)
        /lib64/ld-linux-x86-64.so.2 (0x00007ff70f4f0000)
```

Reply:
i think they should be apt installed here in Dockerfile.opencvsharp-build:

```
FROM mcr.microsoft.com/dotnet/core/aspnet:${PLATFORM_TAG}
WORKDIR /app
COPY --from=build-dotnet-env /opencvsharp/build ./
COPY --from=build-dotnet-env /libOpenCvSharpExtern.so /usr/lib
##### apt install the dependencies here #####
```